README.md

```diff
@@ -15,12 +15,20 @@ To ensure the long-term health of the project, we are temporarily pausing all of
 
 ✅ We will continue reviewing and merging community PRs.
 
+✅ No CI Changes at all - Linting, Testing, e2e testing are all happening on every single pull request.
+
+✅ Security updates, patches, etc. will still be merged to the project (to the `main` branch)
+
```

Comment:
Did you mean the following?

"We still have a security team informed and triaging security issues. Security patches will be merged on a best effort basis (only in the `main` branch)."

Reply:
I don't think we can write up this suggestion, because we have no security team :smile: It feels wrong to say we do when we don't.

Reply:
The readme mentions a security part, which contains a mailing list. Is this not monitored?